demo/app/components/drawer/drawer.component.scss

```diff
@@ -13,3 +13,22 @@
     margin-right: 20px;
   }
 }
+
+.ts-drawer-header{
+  padding-top: 10px;
+  background-color: #000;
+  color: white;
+  width: 100%;
+  height: 60px;
+}
+.ts-drawer-footer {
+  padding-top: 15px;
+  background-color: #000;
+  color: white;
+  width: 100%;
+  height: 50px;
+}
+
+.content {
+  margin-top: 60px;
```

Comment:
Making the consumer responsible for the header/footer spacing is definitely viable. What do you think about handling this for the consumer? (asking because I have a feeling this will be our first request)

Possibly in the header/footer components, each has a getter that returns its size using our window service? Then the drawer component can grab those and set styles to create the header/footer space?